pre_commit/clientlib/validate_config.py

```diff
@@ -71,7 +71,7 @@ def validate_config_extra(config):
             )
         for hook in repo['hooks']:
             try_regex(repo, hook['id'], hook.get('files', ''), 'files')
-            try_regex(repo, hook['id'], hook['exclude'], 'exclude')
+            try_regex(repo, hook['id'], hook.get('exclude', ''), 'exclude')
```

Comment:
In this case it doesn't matter, as long as the fallback value is a valid regex (since this is just the checking code)